ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/AccountDetailsSection.kt

```diff
@@ -0,0 +1,297 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter.views
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.selection.SelectionContainer
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.ContentCopy
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalClipboardManager
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.unit.dp
+import com.revenuecat.purchases.customercenter.CustomerCenterConfigData
+import com.revenuecat.purchases.ui.revenuecatui.BuildConfig
+import com.revenuecat.purchases.ui.revenuecatui.customercenter.CustomerCenterConstants
+import com.revenuecat.purchases.ui.revenuecatui.customercenter.data.getColorForTheme
+
+@Composable
+internal fun AccountDetailsSection(
+    appUserId: String,
+    appearance: CustomerCenterConfigData.Appearance,
+    localization: CustomerCenterConfigData.Localization,
+    modifier: Modifier = Modifier,
+    originalPurchaseDate: String? = null,
+) {
+    if (BuildConfig.DEBUG) {
```

Comment:
I am not 100% sure this will work. This `BuildConfig` is not the app's, it's our own, and I am not sure if it's generated when our SDK is installed. You can double check in an Android project that uses our Maven dependency and see if it's available or not when building on debug vs release.

Reply:
I misunderstood how this worked 😢